File: Domain/Route.cs
namespace Model
{
    public class Routes
    {
        public Guid? RouteId { get; set; }
        public string? StartLocation { get; set; }
        public string? EndLocation { get; set; }

        public int Distance { get; set; }

        public int Duration { get; set; }

    }
}

File: ReservedYouFrontEnd/src/app/models/Route.ts
export class Route {
    routeId:string;
    startLocation:string;
    endLocation:string;
    distance:number;
    duration:number;
    constructor(rId:string,sL:string,eL:string,dst:number,dur:number){
        this.routeId = rId;
        this.startLocation = sL;
        this.endLocation = eL;
        this.distance = dst;
        this.duration = dur;
    }
}